src/js2dt_cli.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+'use strict'
+
+var js2dt = require('./js2dt')
+var program = require('commander')
+
+/**
+ * Callback to write RAML data to console.
+ *
+ * @param  {Error} err
+ * @param  {string} raml
+ */
+function writeToConsole (err, raml) {
+  if (err) {
+    console.log(err)
+    return
+  }
+  console.log('#%RAML 1.0 Library\n')
+  console.log(raml)
+}
+
+/**
+ * Just call js2dt.
+ *
+ * @param  {string} jsonFile
+ * @param  {string} ramlTypeName
+ */
+function js2dtCLI (jsonFile, ramlTypeName) {
```

Comment:
Fixed https://github.com/raml-org/ramldt2jsonschema/commit/e9f20f36909f897d608565ec24b8f48bd3379a8e

Passing JSON as string so input type is identical in dt2js and js2dt - string containing file content.